scripts/run-command-with-timeout.py

```
@@ -0,0 +1,57 @@
+        return
+    except subprocess.TimeoutExpired:
+        try:
+            os.killpg(process.pid, signal.SIGKILL)
+        except ProcessLookupError:
+            return
+        process.wait()
+
+
+def main() -> int:
+    args = parse_args()
+    process = subprocess.Popen(args.command, start_new_session=True)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)